root/defaults/config_local.php

```diff
@@ -1,3 +1,13 @@
+/*
+ ***************************************************
+ * Please read config_default.php for all possible
+ * configuration items.
+ * For changes in config_default.php see CHANGELOG.md
+ * on the upstream project site.
+ * https://github.com/mikespub-org/seblucas-cops/blob/main/config_default.php
+ ***************************************************
+ */
+
```

Comment:
Oops, this is a newline - we just messed up any reply from the web server here :-)

Please move the comment *below* the <?php part as mentioned in the comment